components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -582,4 +583,31 @@ - (void)updateDisplayedTitle {
   _titleLabel.text = [[self class] displayedTitleForTitle:_title style:_style];
 }
 
+- (CGSize)badgeLabelSizeWithText:(NSString *)string {
+  if (string.length <= 0) {
+    return CGSizeZero;
+  }
+
+  NSMutableString *longestAllowableBadgeString = [[NSMutableString alloc] init];
+  __block NSUInteger composedCharacterIndex = 0;
+  [string enumerateSubstringsInRange:NSMakeRange(0, string.length)
+                        options:NSStringEnumerationByComposedCharacterSequences
+                     usingBlock:^(NSString *substring, NSRange substringRange,
+                                  NSRange enclosingRange, BOOL *stop)
+   {
+     [longestAllowableBadgeString appendString:substring];
+     composedCharacterIndex++;
+     if (composedCharacterIndex == kBadgeMaxTextComposedCharacterLength) {
```

Comment:
Please make this `>=` to make sure it terminates.

Reply:
The `*stop = YES;` line tells it to terminate, but I can do a `>=` too